src/gui/systray.h

```diff
@@ -41,6 +42,10 @@ class Systray
     : public QSystemTrayIcon
 {
     Q_OBJECT
+
+    Q_PROPERTY(QString windowTitle READ windowTitle CONSTANT)
+    Q_PROPERTY(bool useNormalWindow READ useNormalWindow CONSTANT)
```

Comment:
Please pay attention because the property type was wrong:
`QString` instead of `bool` and would cause the main dialog to be always a plain old window instead of the nice rounded popup